Util/model_parser/model_parser.H

```diff
@@ -94,18 +98,14 @@ locate(const Real r, const int model_index) {
         loc = ihi;
     }
 
-    return loc;
+    return loc-1;
```

Comment:
What would it mean for locate() to return -1?

Reply:
should be fixed now.